src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/CancellationTokenModelBinder.cs

```diff
@@ -16,7 +17,9 @@ public Task<ModelBindingResult> BindModelAsync(ModelBindingContext bindingContex
         {
             if (bindingContext.ModelType == typeof(CancellationToken))
             {
-                var model = bindingContext.OperationBindingContext.HttpContext.RequestAborted;
+                // Force boxing so we can maintain a reference to this token in validation state.
```

Comment:
Could you clarify this comment? See the `ModelBindingResult` and `ValidationStateDictionary` will store the same reference for the token. But the comment doesn't really make the implication of that consistency clear.